workerbase/lib/build_crosscompiler.sh

```diff
@@ -304,7 +312,7 @@ install_glibc_stage1()
 install_glibc_stage2()
 {
     cd /src/glibc-${glibc_version}_build
-    sudo -E chown buildworker:buildworker -R /src/glibc-${glibc_version}_build
+    sudo -E chown $(id -u):$(id -g) -R /src/glibc-${glibc_version}_build
```

Comment:
While I like this in general, what was the problem here?

Reply:
The alpine image only has `root`.